### PR TITLE
Fix us-south private control plane IPs

### PR DESCRIPTION
**Pull request by bradbehle**
There were quite a few private service endpoint control plane IPs
that were wrong, this PR should fix them.  These policy files were
generated using a tool written by Adam-Mihelcsik and Tamas Soos.  I
made one small change to only use the IPs that the cluster workers
will be using, generated the policies, and then examined them by
hand to make sure all the changes were valid.  They all are valid
and most are because the IPs used for some tugboats in the old policy
were not the ones the cluster workers contact, but instead the ones
that kubectl clients (using the cluster's kubeconfig file) would
contact.  A few others were removed because the carrier6 and carrier118
have been decommissioned.

### CODE REVIEW

File: calico-policies/private-network-isolation/calico-v3/us-south/allow-egress-pods-private.yaml [hunk 1]
```diff
@@ -66,65 +66,60 @@ spec:
   - action: Allow
     destination:
       nets:
-      # Nets for US South: hou02, mex01, sao01, sjc03, sjc04, dal10, dal12, dal13
```
Comment (nirajdp) [inline calico-policies/private-network-isolation/calico-v3/us-south/allow-egress-pods-private.yaml:69]: This comment line was helpful when services teams using kube-sample as reference point raised pr's for their clusters.

File: calico-policies/private-network-isolation/calico-v3/us-south/allow-egress-pods-private.yaml [hunk 2]
```diff
@@ -133,65 +128,60 @@ spec:
...
```
Comment (nirajdp) [inline calico-policies/private-network-isolation/calico-v3/us-south/allow-egress-pods-private.yaml:136]: same comment as before.

**Review by nirajdp — COMMENTED** [review]
Add a nit comment on preserving the comment line as it was helpful info when dealing with clusters in many regions.

**Comment by nirajdp** [comment]
@bradbehle I tested using this PR in us-south and also did one worker reload and all looked fine on our end.

**Review by tamassoos3 — APPROVED** [review]

**Review by derekpoindexter — APPROVED** [review]

File: calico-policies/private-network-isolation/calico-v3/us-south/allow-egress-pods-private.yaml (hunk 1 above)
Comment (bradbehle) [reply calico-policies/private-network-isolation/calico-v3/us-south/allow-egress-pods-private.yaml]: I added the comments back in manually.  We'll have to add that into the tool that creates these as well

**Pull request merged**